websites/migrations/0046_filetype_to_resourcetype.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.1.14 on 2022-04-25 22:37
+
+from django.db import migrations
+
+from websites.constants import WEBSITE_SOURCE_STUDIO
+
+
+def migrate_fields_forward(apps, schema_editor):
+    migrate_field(apps, True)
+
+
+def migrate_fields_backward(apps, schema_editor):
+    migrate_field(apps, False)
+
+
+def migrate_field(apps, forward):
+    WebsiteStarter = apps.get_model("websites", "WebsiteStarter")
+    Website = apps.get_model("websites", "Website")
+    WebsiteContent = apps.get_model("websites", "WebsiteContent")
+    original_name = "filetype" if forward else "resourcetype"
+    updated_name = "resourcetype" if forward else "filetype"
+    try:
+        www_starter = WebsiteStarter.objects.get(name="ocw-www")
+        ocw_www_sites = Website.objects.filter(starter=www_starter)
+        for ocw_www in ocw_www_sites:
+            resources = WebsiteContent.objects.filter(
+                type="resource",
+                website=ocw_www,
+                website__source=WEBSITE_SOURCE_STUDIO,
+                metadata__has_key=original_name,
+            )
+            for resource in resources:
+                value = resource.metadata[original_name]
+                del resource.metadata[original_name]
+                resource.metadata[updated_name] = value
+                resource.save()
+    except Website.DoesNotExist:
+        return
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("websites", "0045_blank_publish_fields"),
+    ]
+
+    operations = [migrations.RunPython(migrate_fields_forward, migrate_fields_forward)]
```

Comment:
Shouldn't one be `migrate_fields_backward`?

Reply:
Wow, yes, I'm not sure how I missed that.